Platforms/QemuQ35Pkg/Plugins/QemuRunner/QemuRunner.py

```diff
@@ -227,11 +227,14 @@ def Runner(env):
         if monitor_port is not None:
             args += " -monitor tcp:127.0.0.1:" + monitor_port + ",server,nowait"
 
+        ## TODO: Save the console mode. The original issue comes from: https://gitlab.com/qemu-project/qemu/-/issues/1674
+        if os.name == 'nt' and qemu_version[0] >= '8':
+            import win32console

```

Comment:
win32console is a part of pywin32, so it needs to be added to the pip requirements:

Example: <https://github.com/microsoft/mu_feature_config/blob/main/pip-requirements.txt?

Reply:
Will do.

Reply:
Updated.